cmd/transcriber/transcribe/output.go

```diff
@@ -4,15 +4,33 @@ import (
 	"fmt"
 	"io"
 	"math"
+	"regexp"
 	"sort"
 	"strings"
 )
 
+var (
+	segmentSanitizationSpacesRE = regexp.MustCompile(`\s+`)
+	// We allow spaces, dots, digits and letters in both ASCII and foreign alphabets.
+	segmentSanitizationSpecialRE = regexp.MustCompile(`[^\s\d\pL\pN.]`)
```

Comment:
We should also allow dashes and underscores. Both are allowed in usernames and dashes are common in real names too.